docs/deep-learning/deep-ssm.md

```diff
@@ -0,0 +1,121 @@
+## ShapeWorks DeepSSM
+DeepSSM is a convolutional neural network (CNN) model that can provide particles directly from unsegmented images after it has been trained. This documentation provides an overview of the DeepSSM process, for a full explanation see: [DeepSSM: A Deep Learning Framework for Statistical
```

Comment:
DeepSSM is the framework which includes data-augmentation and then the prediciton is made by a CNN model trained on image,correspondence pairs.